### PR TITLE
Add simple Makefile for building the runtime and publishing

### PR DESCRIPTION
**Pull request by arichiardi**
Convenient.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.PHONY: publish clean
+
+clean:
+	rm ${RUNTIME_PACKAGE}
+
+${RUNTIME_PACKAGE}:
+ifndef LUMO_BIN_PATH
+	$(error LUMO_BIN_PATH is undefined)
+endif
+	zip -j runtime.zip bootstrap ${LUMO_BIN_PATH} runtime.cljs
```
Comment (grav) [inline Makefile]: Should the env var be used here?

File: README.md [hunk 2]
```diff
@@ -44,6 +44,13 @@ You'll get an error in the end, but an executable will nevertheless be created i
 zip -j runtime.zip bootstrap /path/to/lumo runtime.cljs
 ```
 
+or
+
+```
+make clean (if necessary)
+env LUMO_BIN_PATH=/path/to/lumo make
```
Comment (grav) [inline README.md]: What does the `env` call do here? It seems to work for be by just specifying 

```
LUMO_BIN_PATH=/path/to/lumo make
```

File: README.md [hunk 3]
```diff
@@ -44,6 +44,13 @@ You'll get an error in the end, but an executable will nevertheless be created i
 zip -j runtime.zip bootstrap /path/to/lumo runtime.cljs
```
Comment (grav) [inline README.md]: Maybe we could split up the readme into a "using make" section and a "manual" section?

File: Makefile (hunk 1 above)
Comment (arichiardi) [reply Makefile]: Yes thanks!

File: README.md (hunk 2 above)
Comment (arichiardi) [reply README.md]: It is an alternative that doesn't the same 😁 I can get rid of it

**Comment by arichiardi** [comment]
Done the changes here.

**Comment by arichiardi** [comment]
@grav ping :smile:

**Pull request merged**